pkg/odo/cli/storage/create.go

```diff
@@ -24,49 +25,98 @@ var (
 	`)
 )
 
+const defaultStorageSize = "1Gi"
+
 type StorageCreateOptions struct {
 	storageName      string
 	storageSize      string
 	storagePath      string
 	componentContext string
+
+	devfilePath   string
+	isDevfile     bool
+	componentName string
 	*genericclioptions.Context
 }
 
 // NewStorageCreateOptions creates a new StorageCreateOptions instance
 func NewStorageCreateOptions() *StorageCreateOptions {
-	return &StorageCreateOptions{}
+	return &StorageCreateOptions{devfilePath: "./devfile.yaml"}
```

Comment:
`devfile.yaml` should be using the const from `component` file. In case we were to ever change the name in the future.